watermarker/src/commonTest/kotlin/unitTest/watermarks/TextWatermarkTest.kt

```diff
@@ -68,6 +68,50 @@ class TextWatermarkTest {
         assertEquals(textBytes, content.value)
     }
 
+    @Test
+    fun small_loremIpsum_compression() {
+        val customText = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr"
+        val customTextBytes = customText.encodeToByteArray().asList()
+
+        // Arrange
+        val expectedTrendmark = CompressedRawTrendmark.new(customTextBytes)
```

Comment:
Everything in this block is part of the `// Arrange`. So start with the `// Arrange` comment.

Further, there is a `text` variable already defined in line 30 with its Byte representation (`textBytes`). Check if this can be used either here or in the other test.

Reply:
see [this comment](https://github.com/FraunhoferISST/TREND/pull/99#discussion_r1686937676)